plugins/cost-insights/src/components/CostOverviewCard/CostOverviewCard.test.tsx

```diff
@@ -38,13 +39,30 @@ const mockGroupDailyCost: Cost = {
   trendline: trendlineOf(MockAggregatedDailyCosts),
 };
 
+type TestInputPluginOptions = {
+  showTrendLine: boolean;
```

Comment:
isn't it nicer to make it `hideTrendLine?: boolean;` so it is both optional and defaults to false.